dist/molkobain-datacenter-view/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php

```diff
@@ -0,0 +1,99 @@
+	'Class:Location/Attribute:parent_id' => 'Rodič',
+	'Class:Location/Attribute:parent_id+' => 'Místo, na kterém se nachází (např. pro  \'místnost\', by mělo být \'podlaží\' nebo \'budova\')',
+	'Class:Location/Attribute:locations_list' => 'Podřízená místa',
+	'Class:Location/Attribute:locations_list+' => 'Seznam všech míst zde obsažených',
+	// - PhysicalDevice (default entries for custom classes)
+	'Class:PhysicalDevice/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PhysicalDevice/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'prvek'),
+	'Class:PhysicalDevice/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	// Note: Physical device can be either an host (eg. enclosure) or a device (eg. server), so we use a generic sentence
+	'Class:PhysicalDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:host'], 'prvek', 'jeho hostitel'),
+	// - Rack
+	'Class:Rack/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:Rack/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'rack'),
+	// - Enclosure
+	'Class:Enclosure/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:Enclosure/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'skříň'),
+	'Class:Enclosure/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:Enclosure/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:host'], 'skříň', 'rack'),
+	'Enclosure:baseinfo' => sprintf($aMolkobainThesaurus['Fieldset:baseinfo']),
+	'Enclosure:moreinfo' => sprintf($aMolkobainThesaurus['Fieldset:moreinfo']),
+	'Enclosure:otherinfo' => sprintf($aMolkobainThesaurus['Fieldset:otherinfo']),
+	'Enclosure:dates' => sprintf($aMolkobainThesaurus['Fieldset:dates']),
+	// - Datacenter device
+	'Class:DatacenterDevice/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:DatacenterDevice/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'zařízení'),
+	'Class:DatacenterDevice/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'zařízení'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'zařízení'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'napájení'),
+));
+
+// UI
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```

Comment:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

Same as the previous one.